session/session.go

```diff
@@ -64,16 +82,77 @@ func (s *Session) Has(key string) bool {
 	return val != nil
 }
 
+func (s *Session) Invalidate() error {
+	s.Flush()
+	return s.Migrate(true)
+}
+
+func (s *Session) Migrate(destroy ...bool) error {
+	shouldDestroy := false
+	if len(destroy) > 0 {
+		shouldDestroy = destroy[0]
+	}
+
+	if shouldDestroy {
+		err := s.driver.Destroy(s.GetID())
+		if err != nil {
+			return err
+		}
+	}
+
+	s.SetID(s.generateSessionID())
+
+	return nil
+}
+
+func (s *Session) Missing(key string) bool {
+	return !s.Exists(key)
+}
+
+func (s *Session) Only(keys []string) map[string]any {
+	return supportmaps.Only(s.attributes, keys...)
+}
+
+func (s *Session) PreviousUrl() string {
```

Comment:
In what scenarios is the function used, please?

Reply:
`PreviousUrl()` can be used to retrieve the URL of the previous request made by the user in the current session. It is useful when we need to redirect the user back to the previous request after performing some kind of action.

Reply:
Yes, I got you, I remembered a few scenarios about this, but I think the function currently has no practical use in Goravel, compared with Larave, Goravel still lacks a lot in terms of the View module. In conclusion, we can leave it here, it doesn't matter.

Reply:
should I remove it?

Reply:
Of course, it can be removed if you also agree.

Reply:
Since, we don't need it right now, I will remove it.